### PR TITLE
Feature/content

### PR DESCRIPTION
**Pull request by momofcats**
Add main content to product.html. I made the picture and the card as separate blocks. I'm not sure if they are supposed to be blocks or just elements of content

### CODE REVIEW

File: product.html [hunk 1]
```diff
@@ -1,15 +1,38 @@
...
+		<link rel="stylesheet" href="blocks/product.css" />
+		<title>Product</title>
+	</head>
+	<body class="page">
+		<ul class="nav">
+			<li><img src="images/arrow.svg" alt="go back icon" /></li>
+			<li><img src="images/star.svg" alt="star-icon" /></li>
+		</ul>
+		<main class="content">
+			<img class="image" src="images/macaron.png" />
```
Comment (kvnloughead) [inline product.html:15]: You should change the file name to macaroon.png

File: product.html [hunk 2]
```diff
@@ -1,15 +1,38 @@
...
+			<div class="item">
+				<h2 class="item__title">Pink Macaroon</h2>
+				<div class="item__price-row">
+					<div class="item__buttons">
+						<button class="item__button item__button_type_minus" type="button"> 
+						</button>
+						<p class="item__quantity">1</p>
```
Comment (kvnloughead) [inline product.html:22]: item__quantity and item__price should have font-weight bold.

File: product.html (hunk 1 above)
Comment (momofcats) [reply product.html:15]: done

File: product.html (hunk 2 above)
Comment (momofcats) [reply product.html:22]: thanks, somehow I missed it

**Pull request merged**